src/main/java/org/elasticsearch/index/merge/scheduler/ConcurrentMergeSchedulerProvider.java

```diff
@@ -172,7 +172,9 @@ public void onRefreshSettings(Settings settings) {
                 logger.info("updating [{}] from [{}] to [{}]", MAX_MERGE_COUNT_KEY, ConcurrentMergeSchedulerProvider.this.maxMergeCount, maxMergeCount);
```

Comment:
Hmm we should probably subtract one fro the maxMergeCount we log here (since we added one on init and update)?

Likewise `if (maxMergeCount != ConcurrentMergeSchedulerProvider.this.maxMergeCount) {` should subtract 1?